react/MonthPicker/CustomMonthPicker/CustomMonthPicker.js

```diff
@@ -141,8 +141,10 @@ export default class CustomMonthPicker extends Component {
     const monthValue = String(month || '');
     const yearValue = String(year || '');
 
+    const sharedProps = {
+      onBlur: this.handleBlur
```

Comment:
There are quite a few shared props that aren't apart of this object, would it make sense to add them here? I think it might be more performant to just add these directly to the `Dropdown`s below.